cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,312 @@
+"""Manual tests"""
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [test_requirements.v2v, pytest.mark.manual]
+
+
+@pytest.mark.tier(2)
+def test_osp_test_flavors_can_be_selected_creating_migration_plan():
+    """
+    title: OSP: Test flavors can be selected creating migration plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan amd choose flavors
+            3. Start migration
+        expectedResults:
+            1. Mapping created and visible in UI
+            2. Plan created and visible in UI
+            3. Successful migration from vmware to OSP with the selected flavor
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_creating_multiple_migration_plans_with_same_name():
```

Comment:
What is the reason for the second `test` in all of these test case names? Why not name this `test_osp_creating_multiple_migration_plans_with_same_name`?

Reply:
@john-dupuy : copy pasted from test_manual.py  :smile: 
Going to automate these soon